src/index.html

```diff
@@ -21,7 +21,16 @@ <h1 class="post-title fw-500">
           </a>
         </h1>
 
-        <span class="post-date">{{ post.date | date_to_string }}</span>
+        <div class="d-flex align-items-center mb-4 text-muted">
```

Comment:
If this whole newly added block duplicated, we might want to look into moving it to an include to reduce duplication. I'm going to take a stab at it later.